src/pages/experiments/[experimentId]/data-processing/data-integration/components/CalculationConfig.jsx

```diff
@@ -61,11 +61,20 @@ const CalculationConfig = () => {
     <>
       <Space direction='vertical' style={{ width: '100%' }} />
       <Form size='small'>
+        {changesOutstanding && (
+          <Form.Item>
+            <Alert
+              message='Your changes are not yet applied. To update the plots, click Apply.'
+              type='warning'
+              showIcon
+            />
+          </Form.Item>
+        )}
```

Comment:
These changes were made to bring Data Integration more in line with what we have now with Configure Embedding.